test/services/DataGeneratorSpec.scala

```diff
@@ -176,15 +216,15 @@ class DataGeneratorSpec extends PlaySpec with CSVTestData with GuiceOneAppPerSui
   }
 
   "generateRowData" should {
```

Comment:
We're already checking to see if generareRowData handles a file with and without errors, but not if `ErsValidator.validateRow` fails itsef.

Reply:
I'll add a test for that